tools/unit/src/main/kotlin/com/datadog/tools/unit/ReflectUtils.kt

```diff
@@ -46,17 +46,7 @@ inline fun <reified T, R> Class<T>.setStaticValue(
     // make it accessible
     field.isAccessible = true
```

Comment:
do we need this line? it is already there in the `setFieldValue`

Reply:
Yes because those are different fields:
1. Is the field in the `T` class for which we want to set the value
2. Is the field `accessFlags` or `modifiers` in the `Field.class`